.azure-pipelines/template-test-minimal-conda-env-install.yml

```diff
@@ -0,0 +1,140 @@
+# File: template-test-minimal-conda-env-install.yml
+parameters:
+  - name: environment
+    type: string
+    default: prod
+    values:
+      - prod
+      - dev
+  - name: envToRepo
+    type: object
+    default:
+      all:
+        - geology
+        - geophysics
+      all_deps_only:
+        - geology
+        - geophysics
+      geology: geology
+      geophysics: geophysics
+      interop: public
+      simpeg: geophysics
+
+stages:
+  - stage: TestInstallationOnMinimalCondaEnv
```

Comment:
**question:** does it follows the naming convention in place, if any?

(I admit I am somewhat confused as this is an Azure pipeline, and I am used to see the ones of github)

Reply:
Microsoft is pretty annoying with its documentation. If there is no official guideline, they use:
- `camelCase` for variable like stage, dependsOn, variable etc
- `PascalCase` for job name and stage name

Reply:
the exemples are the guidelines then, ok... 
jobs and stages follow some sort of class naming convention, and the rest variable naming convention.
Anyway, as long as you pick a naming convention consistently, that is fine.
Thanks